incubator/hnc/internal/validators/object_test.go

```diff
@@ -102,13 +116,7 @@ func TestInheritedFromLabel(t *testing.T) {
 		name:     "Label is added",
 		newLabel: api.LabelInheritedFrom, newValue: "foo",
 		fail: true,
-	}, {
```

Comment:
This case is removed because it's handled in the `Handle()` func but not `handle()` func now?

Reply:
Yup, exactly.